plugin/trino-resource-group-managers/src/main/java/io/trino/plugin/resourcegroups/db/ResourceGroupsDao.java

```diff
@@ -23,19 +23,19 @@
 public interface ResourceGroupsDao
 {
     @SqlUpdate("CREATE TABLE IF NOT EXISTS resource_groups_global_properties (\n" +
-            "  name VARCHAR(128) NOT NULL PRIMARY KEY,\n" +
-            "  value VARCHAR(512) NULL,\n" +
-            "  CHECK (name in ('cpu_quota_period'))\n" +
+            "  \"name\" VARCHAR(128) NOT NULL PRIMARY KEY,\n" +
+            "  \"value\" VARCHAR(512) NULL,\n" +
```

Comment:
This is "maybe OK change" only because this production-looking code is not used for production.
Before the change, we were creating "probably uppercase" column names (unless PostgreSQL), now we're creating lowercase column names, which would break queries.

As a prep commit, we should move this queries to test resources, make it clear they are for h2 only, eg by placing it inside `io.trino.execution.resourcegroups.db.H2TestUtil` class, and then it's "whatever works".

Reply:
Actually I think that it's used in production via `DbResourceGroupsModule` which allows arbitrary jdbc implementation

Reply:
if this is used in produciton... what is the resolution then?

Reply:
I've used this option to allow key and value as correct identifiers. No changes here.